packages/devtools/lib/src/timeline/timeline_protocol.dart

```diff
@@ -322,7 +322,14 @@ class TimelineFrame {
   final String id;
 
   /// Marks whether this frame has been added to the timeline.
-  final Completer<Null> addedToTimeline = Completer();
+  ///
```

Comment:
nit: rebase so this change doesn't show up in this pull request.

Reply:
odd, i tried to rebase but I must have messed up somewhere. Good to go now.